charts/cass-operator-chart/templates/customresourcedefinition.yaml

```diff
@@ -6373,7 +6643,6 @@ spec:
           - serverType
           - serverVersion
           - size
-          - storageConfig
```

Comment:
I think that this is wrong - we should not change the defaults. By default, the DC definition must have a storageConfig, and if the user wants to he can override this per rack.